module/documents/actor.js

```diff
@@ -246,6 +219,21 @@ export default class Actor5e extends Actor {
 
   /* -------------------------------------------- */
 
+  /**
+   * Initialize derived AC fields for Active Effects to target.
+   * Mutates the system.attributes.ac object.
+   * @private
+   */
+  _prepareBaseArmorClass() {
```

Comment:
A consideration: in the main foundry VTT code base we are moving towards using true private methods like `#prepareBaseArmorClass` in cases where we feel the method can/should be private, otherwise treating methods as `@protected` if they are underscored. Might that be a convention we want to employ here as well?